src/main/java/address/controller/MainController.java

```diff
@@ -363,15 +361,13 @@ public void releaseResourcesForAppTermination(){
         browserManager.freeBrowserResources();
     }
 
-    public void loadGithubProfilePage(String githubUserName){
-        browserManager.getBrowser().loadURL("https://www.github.com/" + githubUserName);
+    public void loadGitHubProfilePage(Person person){
+        browserManager.loadGitHubProfilePage(person);
```

Comment:
Maybe we can pass all the persons needed to be loaded here instead of coupling `browserManager` with `modelManager`.